### ISSUE TITLE
Images doesn't show in preview

### ISSUE THREAD

**Issue by Kiiv** [grosenberg/fluentmark#14]
I'm trying to insert an image in my documentation. The path is correct and the declaration too (in the generated doc the image shows correctly). But in the preview I got an empty picture just like if the file was not found.

To reproduce, insert an image like this :
```
#### TestImage
[![My image](images/myImage.png)](images/myImage.png)

```

I'm using Pandoc converter but it seems the same with all others.

**Comment by grosenberg** [comment]
Is the file `images/myImage.png` within the same project containing the `.md` file?

The Pandoc HTML converter may be having trouble with the compound formatting -- that is what the previewer would be using. 

What happens if you try

    ![My image](images/myImage.png)

or 

    ![My image](./images/myImage.png)

**Comment by Kiiv** [comment]
Hi !

Yes image is in the same project.
Other converter has the same problem so it's not pandoc.
Tried under Windows and had the same issue.

**Comment by grosenberg** [comment]
Fixed with v0.12.0.
